src/main/cpp/t08_cycleswap.cpp

```diff
@@ -19,5 +19,20 @@
 using namespace std;
 
 int t08_cycleswap() {
-
+  int qua;
+  cin >> qua;
+  int *arr = new int[qua];
+  int swapper = 0;
+  int n = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'n' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4739147)